sunpy/meta/meta.py

```diff
@@ -0,0 +1,189 @@
+from __future__ import absolute_import
+
+import numpy as np
+from astropy.units import Quantity
+
+__all__ = ['MetaItem', 'Meta']
+
+
+class Meta(object):
```

Comment:
This object should subclass from the `collections.MutableMapping` abstract base class: https://docs.python.org/3.6/library/collections.abc.html